### PR TITLE
📄 Signal description copy change

### PR DESCRIPTION
**Pull request by Ken-tech-max**
- [ ]  #3091 
- [ ] ~~#2601~~
- [ ] ~~#2770~~

### CODE REVIEW

File: packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ export const StakingPolicyAndReward = () => {
   const { watch } = useFormContext()
   const leavingUnstakingPeriod = watch('stakingPolicyAndReward.leavingUnstakingPeriod')
 
+  const maxAllowedForStakingAmount = 200
```
Comment (WRadoslaw) [inline packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx:16]: This validation must be based on node constant

**Review by WRadoslaw — CHANGES_REQUESTED** [review]
Hello! Since whole validation was transferred to React Hook Form, there is no need to additionally keep it in `useState`. This issue can be resolved by only changing error message inside AddNewProposalModal `schemaFactory` function. Validation is working correctly (returns valid when value is equal), so message is only thing tthat's need to be changed.

**Comment by Ken-tech-max** [comment]
Fixed : 3091

**Comment by Ken-tech-max** [comment]
Updated issue 2770

**Comment by singulart** [comment]
@Ken-tech-max Any updates on this?

**Comment by Ken-tech-max** [comment]
I am working on it
Just started right now

**Comment by dmtrjsg** [comment]
@singulart is it possible to clarify which issues exactly were fixed with this PR? Possible to commit them separately?

cc @thesan

**Comment by singulart** [comment]
The worker doesn't respond, it's not realistic to extract the info requested...

**Review by thesan — APPROVED** [review]

**Review by WRadoslaw — APPROVED** [review]

**Pull request merged**